docs/ci.md

```diff
@@ -75,9 +75,12 @@ package:
 One way to achieve this is to set the `NPM_TOKEN` in the CI environment, and from a script do:
 
 ```bash
-echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
+npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN
 ```
 
+This will create/update the `.npmrc` file and add the token there. Ideally you should either `.gitignore` this file, 
+otherwise you will end up committing it to your repo if you are using release-it's git options.
```

Comment:
It's not entirely correct, only when using `addUntrackedFiles: true`:

```suggestion
otherwise you might end up committing it to your repo if you are using release-it's `git` options.
```

Reply:
True! Thanks for pointing it out.